product_docs/docs/biganimal/release/getting_started/identity_provider/index.mdx

```diff
@@ -34,36 +34,42 @@ http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress
-
+2. In the **Configure SAML response** section, configure your identity provider to send the SAML response. The SAML response must include the following attributes:
+   - `givenname`: BigAnimal uses the value as the given name in the profile of the authenticated user.
+   - `surname`: BigAnimal uses the value as the surname in the profile of the authenticated user.
+   - `name`: BigAnimal uses the value as the full name (`givenname` joined with `surname`) in the profile of the authenticated user.
+   - `emailaddress` (optional): BigAnimal uses the value for the email address in the profile of the authenticated user. 
+   
    Provide a `NameID` element in the `Subject` element. Provide your email ID as the value to `NameID`. BigAnimal uses the email ID you provide as your username and primary email, so format `NameID` like an email address.
 
    For example:
 
    ![](../images/nameID.png)
-3. On the **Setup Configuration** tab, enter the configuration information for your preferred SAML identity provider:
+3. In the **SAML settings** section, enter the configuration information for your preferred SAML identity provider:
 
    | Field      | Description |
    | ---------- | ----------- |
-   | Issuer URL | URL string uniquely associated with your identity provider partner. |
    | Single Sign-On URL | The identity provider's sign-on URL. |
-   | Identity provider signature certificate | Identity provider's assertion signing certificate (`.cer` or `.cert`). Coordinate with your identity provider partner to obtain this certificate securely. |
+   | Identity Provider Signature Certificate | Identity provider's assertion signing certificate (`.cer` or `.cert`). Coordinate with your identity provider partner to obtain this certificate securely. |
    | Request Binding | SAML Authentication Request Protocol binding used to send the authentication request: HTTP-Redirect,  HTTP-Post, or Hybrid (SAML request is REDIRECT and response is POST). |
    | Response Signature Algorithm (RSA) Algorithm | The signature algorithm used to sign the SAML AuthNRequest (RSA SHA-1 or RSA SHA-256). |
 4. Select **Test Connection**. If you connect to your identity provider successfully, your identity provider's login screen appears. If an error message appears, contact [Support](/biganimal/latest/overview/support).
 
 Once your identity provider is set up, you can view your connection status, ID, login URL, audience URI, and assertion consumer service URL from the BigAnimal portal on the Identity Provider page. Select **Admin > Identity Provider** to access it.
 
+## Add a domain
+
+A verified domain is necessary so that your users can sign into their BigAnimal accounts. 
+
+In the **Domains** tab, enter the domain and select **Next: Verify Domain**. Copy the TXT record and follow the instructions. Select **Done**. 
+
+Your domain and its status appears on the **Domains** tab, where you can delete or verify it. To add another domain, select **Add Domain**. 
```

Comment:
we also need to mention something like 
"only in the case you have at least one verified domain, your IdP's status can become `Active`. The identity provider's status change caused by the domain verification status can take up to 3 minutes" 

<img width="980" alt="image" src="https://user-images.githubusercontent.com/79249661/232464586-30ec4e66-cb7c-4718-8b90-bd2654e52b6e.png">

Reply:
let me know if my above suggestion is vague to you @ccestes

Reply:
I wrote this as: 

> When you have at least one verified domain, the identity provider status becomes **Active** on the the **Identity Providers** tab. The identity provider status can take up to three minutes to update after verifying a domain.

Reply:
(With the last sentence as its own note)

Reply:
Hi @ccestes my fault I didn't explain the entire scenario. 

If the user loses its last verified domain, the status of the IdP would become "Inactive". This will also take up to 3 minutes.

Reply:
Do you think we need to mention this as well? ☝️

Reply:
I can update it to this: 

> When you have at least one verified domain, the identity provider status becomes **Active** on the **Identity Providers** tab. The status becomes **Inactive** when the last verified domain is no longer verified.